### PR TITLE
Implemented the notion details page

### PR DESCRIPTION
**Pull request by once-upon-a-tam**
Its data is mocked for the moment, as we don't have a way to create nor retrieve real data.
You can test this PR using the `/notions/1` url.

### CODE REVIEW

**Comment by once-upon-a-tam** [comment]
[As a developer, I want to have the shell of a notion's details page.](https://trello.com/c/aJmnkji3/48-as-a-developer-i-want-to-have-the-shell-of-a-notions-details-page)

File: src/components/App.jsx [hunk 1]
```diff
@@ -78,19 +78,30 @@ const App = () => (
 			<LabelComponent>Default label component</LabelComponent>
 			<LabelComponent textStyle="bodycopy">BodyCopy label component</LabelComponent>
 		</StyledDiv>
+		<StyledDiv position="relative" display="flex" flexDirection="column" gridGap=".5em">
+			<LabeledCheckbox id="lcb1">Labeled checkbox</LabeledCheckbox>
```
Comment (frenchykiller) [inline src/components/App.jsx:82]: l.81 à l.86 obsolètes

File: src/components/App.jsx [hunk 2]
```diff
@@ -78,19 +78,30 @@ const App = () => (
...
+			<LabeledCheckbox checked id="lcb2">Default checked labeled checkbox</LabeledCheckbox>
+			<LabeledCheckbox disabled id="lcb3">Disabled labeled checkbox</LabeledCheckbox>
+			<LabeledCheckbox disabled checked id="lcb4">Disabled default checked labeled checkbox</LabeledCheckbox>
+		</StyledDiv>
 		<StyledDiv position="relative">
-			<FieldsetComponent title="Basic checkbox components">
+			<FieldsetComponent title="Basic fieldset component" display="flex" flexDirection="column" gridGap=".5em">
```
Comment (frenchykiller) [inline src/components/App.jsx:88]: `title` devrait être "Basic checkbox component"

File: src/components/App.jsx [hunk 3]
```diff
@@ -78,19 +78,30 @@ const App = () => (
...
 				<LabeledCheckbox id="lcb5">Labeled checkbox</LabeledCheckbox>
 				<LabeledCheckbox labelTextStyle="bodycopy" checked id="lcb6">Default checked labeled checkbox</LabeledCheckbox>
 				<LabeledCheckbox disabled id="lcb7">Disabled labeled checkbox</LabeledCheckbox>
 				<LabeledCheckbox disabled checked id="lcb8">Disabled default checked labeled checkbox</LabeledCheckbox>
 			</FieldsetComponent>
 		</StyledDiv>
+		<StyledDiv position="relative" display="flex" flexDirection="column" gridGap=".5em">
```
Comment (frenchykiller) [inline src/components/App.jsx:95]: l.95 à l.99 obsolètes

File: src/components/App.jsx [hunk 4]
```diff
@@ -78,19 +78,30 @@ const App = () => (
...
+			<LabeledRadioButton name="rg1" id="lrb1">Labeled radio button</LabeledRadioButton>
+			<LabeledRadioButton name="rg1" checked id="lrb2">Default checked labeled radio button</LabeledRadioButton>
+			<LabeledRadioButton name="rg1" disabled id="lrb3">Disabled labeled radio button</LabeledRadioButton>
+		</StyledDiv>
 		<StyledDiv position="relative">
-			<FieldsetComponent title="Basic radio button components">
-				<LabeledRadioButton name="rg2" id="lrb5">Labeled radio button</LabeledRadioButton>
-				<LabeledRadioButton labelTextStyle="bodycopy" name="rg2" checked id="lrb6">Default checked labeled radio button</LabeledRadioButton>
-				<LabeledRadioButton name="rg2" disabled id="lrb7">Disabled labeled radio button</LabeledRadioButton>
+			<FieldsetComponent title="Basic fieldset component" display="flex" flexDirection="column" gridGap=".5em">
```
Comment (frenchykiller) [inline src/components/App.jsx:101]: `title` devrait être "Basic radio button components"

File: src/theme/formStyles.js [hunk 5]
```diff
@@ -95,26 +97,17 @@ const radioButtonStyle = {
...
-		backgroundColor: jasmine.darker1,
-		'&::before': {
-			content: '""',
-			borderRadius: '50%',
-			height: '8px',
-			width: '8px',
-			margin: '3px',
-			backgroundColor: (({ disabled }) => (disabled ? jasmine.darker1 : coconut.default)),
-		},
+		backgroundColor: parsnip.lighter1,
```
Comment (frenchykiller) [inline src/theme/formStyles.js:100]: attention au hover d'un radio button disabled

File: tests/components/notions/QuestionForm.test.js [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+			expect(wrapper).toMatchSnapshot();
+		});
+	});
+
+	describe('answer selection change', () => {
+		it('should replace the answer if singleChoice is set to true.', () => {
+			const onSubmit = jest.fn();
+			render(<QuestionForm answers={answers} singleChoice onSubmit={onSubmit} />);
+
+			fireEvent.click(screen.getAllByRole('radio')[0]);
```
Comment (frenchykiller) [inline tests/components/notions/QuestionForm.test.js]: utiliser userEvent.click au lieu de fireEvent.click

**Review by frenchykiller — CHANGES_REQUESTED** [review]

**Review by frenchykiller — APPROVED** [review]

**Pull request merged**